cpp/server/json_output.h

```diff
@@ -0,0 +1,41 @@
+#ifndef CERT_TRANS_SERVER_JSON_OUTPUT_H_
+#define CERT_TRANS_SERVER_JSON_OUTPUT_H_
+
+#include <string>
+
+#include "base/macros.h"
+
+struct evhttp_request;
+class JsonObject;
+
+namespace cert_trans {
+namespace libevent {
+class Base;
+}  // namespace libevent
+
+
+static const char kJsonContentType[] = "application/json; charset=ISO-8859-1";
```

Comment:
This could be moved to the ```.cc``` wholesale, I think? If not, move the definition?

And "ISO-8859-1"? What is this, the 90's? UTF-8! :wink: